app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/FractionInteractionViewModel.kt

```diff
@@ -20,7 +20,7 @@ import org.oppia.android.util.math.FractionParser
 import javax.inject.Inject
 
 /** [StateItemViewModel] for the fraction input interaction. */
-class FractionInteractionViewModel private constructor(
+class FractionInteractionViewModel public constructor(
```

Comment:
why do we need to modify this?

```
By default, all constructors are public , which effectively amounts to them being visible everywhere the class is visible
```

https://kotlinlang.org/docs/classes.html#constructors

Reply:
Do you mean that we need to remove the access modifier since all constructors are public by default?